### ISSUE TITLE
Ypres - Bug crossing servers.

### ISSUE THREAD

**Issue by KytaSaeso** [Dynarev/mythmoor#4]
Sorry to bother u guys, now how busy u get, but might have a bug. I butchered a mimic in Moors, came from NL, and the result was that from a Pink Uni, not the moonstone norm? It belonged to Tiralem

**Comment by KytaSaeso** [comment]
I assume this is te same problem with Database ID's as with carts crossing servers, but thought it better to place here for review.

**Comment by Dynarev** [comment]
Yes I have one other thing I need to look into for this, the stored generated template id's in the mod database, I may be able to create a swap to proper by copying the mod db from prime to northlands, I will compare them and see what it looks like.
